bridge_adaptivity/module/mixins.py

```diff
@@ -97,3 +97,12 @@ class CollectionMixin(object):
     def get_queryset(self):
         qs = Collection.objects.filter(owner=self.request.user)
         return qs
+
+
+class BackURLMixin(object):
+    def get_context_data(self, **kwargs):
```

Comment:
probably it should be better to define field name which should be used for back button.
in some cases it could be the same as used in the form and in this case we will change it in the view.

```
class BackURLMixin(object):
    back_url_field_name = 'back_url'

    def get_context_data(self, **kwargs):
        context = super(BackURLMixin, self).get_context_data(**kwargs)
        back_url = self.request.GET.get(self.back_url_field_name)
        if back_url:
            context['back_url'] = back_url
        return context
```

Reply:
It is not clear enough what is the pros in proposed change?